frontend/src/components/Login/AccountCreation.tsx

```diff
@@ -1,66 +1,65 @@
-import React, { ReactElement, useState } from "react";
-import styled from "styled-components";
+import React, { ReactElement, useState } from 'react';
+import styled from 'styled-components';
 
 const CloseButton = styled.button`
-    float: right;
-    margin-bottom: 5px;
-    position: absolute;
-    left: 89%;
-    bottom: 85%;
+  float: right;
+  margin-bottom: 5px;
+  position: absolute;
+  left: 89%;
+  bottom: 85%;
 `;
 
 interface Props {
-    closeModal: any;
+  closeModal: any;
```

Comment:
If you hover over the setter function returned from useState you'll get the signature function for this type if you don't want to use any.

Reply:
Also if your function doesn't return anything and it doesn't take any args you could use () => void